samples/ActionConstraintSample.Web/project.json

```diff
@@ -12,9 +12,12 @@
     "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-*"
   },
   "frameworks": {
-    "net451": { },
-    "dnxcore50": {
-      "imports": "portable-net451+win8",
+    "net451": {},
+    "netstandardapp1.5": {
+      "imports": [
+        "dnxcore50",
+        "portable-net451+win8"
+      ],
```

Comment:
So the portable bits are here because there's an external dependency in the chain that doesn't directly target `dnxcore50` or `netstandardapp1.5` but does target something compatible with the `portable-net451+win8`.

As for why we have the `dnxcore50` in the imports now instead of at the top level: This is a temporary stop-gap to enable us to work with packages that haven't 100% transitioned to netstandard yet. Aka, if a package supports netstandard it'll use the netstandard target, if not it falls back to the imports.